reference/resources/digital_wallet.yaml

```
@@ -0,0 +1,62 @@
+---
+title: Digital Wallet
+type: object
+description: |-
+  A digital wallet (e.g. Apple Pay) that has been registered.
+x-tags:
+  - Digital Wallets
+
+properties:
+  type:
+    type: string
+    example: digital-wallet
+    description: "`digital-wallet`."
+    enum:
+      - digital-wallet
+
+  provider:
+    type: string
+    example: apple
+    description: The name of the digital wallet provider.
+    enum:
+      - apple
+
+  id:
+    type: string
+    example: 8d3fe99b-1422-42e6-bbb3-932d95ae5f79
+    description: The ID of the registered digital wallet.
+    format: uuid
+
+  merchant_name:
+    type: string
+    example: Gr4vy
+    description: The name of the merchant the digital wallet is registered to.
+
+  merchant_url:
+    type: string
+    example: "https://gr4vy.com"
```

```suggestion
    example: "https://example.com"
```

Setting this to gr4vy.com might be a bit confusing as it needs to be the merchant's URL not ours